contracts/casino/include/casino/casino.hpp

```diff
@@ -179,6 +188,17 @@ class [[eosio::contract("casino")]] casino: public eosio::contract {
     [[eosio::action("rmgamenobon")]]
     void remove_game_no_bonus(name game_account); // rm game from bonus restricted games table
 
+    // ==========================
+    // token
+    [[eosio::action("addtoken")]]
+    void add_token(std::string token_name);
```

Comment:
can it be const std::string& ? or it meaningless

Reply:
It's better to write `const std::string&` (avoids extra copy). Since performance is not an issue in smart contracts I try to keep it as simple as possible.